website/tables/aws/aws_emr_notebook_executions.md

```diff
@@ -0,0 +1,38 @@
+# Table: aws_emr_notebook_executions
+
+This table shows data for Amazon EMR Notebook Executions.
+
+https://docs.aws.amazon.com/emr/latest/APIReference/API_NotebookExecution.html
+
+The composite primary key for this table is (**cluster_arn**, **arn**).
```

Comment:
Isn't the `arn` alone enough in this case? Why include `cluster_arn` in the PK as well?

Reply:
Yeah, I think keeping `cluster_arn` but with PK set to false should do the trick 👍